src/python/pants/util/strutil_test.py

```diff
@@ -84,9 +84,9 @@ def test_strip_chroot_path() -> None:
         )
         == dedent(
             """\
-        Would reformat src/python/example.py
-        Would reformat test.py
-        Would reformat custom_tmpdir.py
+        Would reformat /<sandbox-1>/src/python/example.py
+        Would reformat /<sandbox-2>/test.py
+        Would reformat /<sandbox-3>/custom_tmpdir.py
```

Comment:
I'd like to see a test case with multiple paths from the same sandbox to show that the counting works as intended.

Reply:
It already did in the test for greedy matching, but added a few more lines here as well.